components/director/internal/domain/webhook/service.go

```diff
@@ -48,11 +49,7 @@ func NewService(repo WebhookRepository, appRepo ApplicationRepository, uidSvc UI
 }
 
 func (s *service) Get(ctx context.Context, id string) (*model.Webhook, error) {
-	tnt, err := tenant.LoadFromContext(ctx)
-	if err != nil {
-		return nil, err
-	}
-	webhook, err := s.webhookRepo.GetByID(ctx, tnt, id)
+	webhook, err := s.webhookRepo.GetByIDGlobal(ctx, id)
```

Comment:
with this now i can delete a webhook for app that belongs to a tenant different from mine (similar to the update security issue above)